my-profile-webapp/src/main/webapp/my-app/lec/services.js

```diff
@@ -41,6 +41,24 @@ define(['angular'], function(angular) {
              });
         };
         
+        var getEmergencyPhoneNumber = function(){
+            return $http.get('/profile/api/emergencyPhoneNumber/get.json').success(
+                function(data, status){
+                    return data;
+                }).error(function(data, status){
+                    miscService.redirectUser(status, "Get emergency contact info");
```

Comment:
Looks like [miscService uses the caller parameter](https://github.com/UW-Madison-DoIT/uw-frame/blob/128c47ca85a392b4f2f1c8771d3aae225379219e/src/main/webapp/portal/misc/services.js#L16) under some circumstances to inform a console log message.  If we're continuing to care about the identity of the caller, should update that parameter on this method call to maybe "getEmergencyPhoneNumber" or so.  Ditto the call below on line 58.